src/mpl/src/gpu/mpl_gpu_ze.c

```diff
@@ -52,6 +52,42 @@ static uint32_t local_ze_device_count;  /* Counts all local devices and subdevic
 static uint32_t global_ze_device_count; /* Counts all global devices and subdevices */
 static int max_dev_id;          /* Does not include subdevices */
 static int max_subdev_id;
+static int global_dev_count;
+static int global_subdev_count;
```

Comment:
Would recommend different names; these don't actually refer to global values since they are based on locally-defined `max_*_id`